blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -35,6 +36,14 @@ where ``column_constraint`` is::
                             FULLTEXT [ WITH ( analyzer = analyzer_name ) ]  }
     }
 
+where ``reroute_type`` is::
+
+    { MOVE table_ident SHARD shard_id FROM node_id TO node_id
```

Comment:
why is there `table_ident` mentioned again. wouldn't result this into something like
```
ALTER TABLE table_ident MOVE table_ident SHARD shard_id FROM node_id TO node_id2;
```